src/main/webapp/forum.html

```diff
@@ -52,6 +52,11 @@ <h2>Don't have an account?</h2>
       </div>
     </div>
     <div id="forum">
+      <div id="search">
+        <h2>Search:</h2>
+        <input type="text" id="search-input"></input>
+        <button id="search-button"><i class="fa fa-search"></i></button>
```

Comment:
Do `fa` and `fa-search` do anything? Remove if not

Reply:
They are icon classes that I use to make a magnifying glass icon.

Reply:
I think they stand for font awesome

Reply:
Am I allowed to use outside icons?